contract/create_asset_request.go

```diff
@@ -0,0 +1,74 @@
+package contract
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/vkhichar/asset-management/domain"
+)
+
+var checkStatus = [3]string{"active", "retired", "undermaintenance"}
```

Comment:
make this constant . define userdefined type as AssetStatus like type AssetStatus string, then add all values that aseetStatus can have